README.md

```diff
@@ -1315,8 +1318,76 @@ features: {
 
 Events published by the widget. Subscribe to these events using [on](#onevent-callback-context).
 
+### ready
+
+Triggered when the widget is ready to accept user input for the first time. Returns a `context` object containing the following properties:
+
+- **controller** - Current controller name
+
+```javascript
+signIn.on('ready', function (context) {
+  if (context.controller === 'primary-auth') {
+    // The primary authentication form is ready for user input
+  }
+});
+```
+
+### afterError
+
+The widget will handle most types of errors - for example, if the user enters an invalid password or there are issues authenticating. To capture an authentication state change error after it is handled and rendered by the Widget, listen to the `afterError` event. For other errors types, it is encouraged to handle them using the [`renderEl` error handler](#renderel).
```

Comment:
A developer unfamiliar with the Widget may not know what an authentication state change error is. I think we should keep this description simple, like

> To get notified when a recoverable error occurs (for example, the user enters an invalid password), subscribe to the `afterError` event.

Talking about the renderEl error handler here is likely to be confusing.

Reply:
Makes sense. Typo nit: **errors**